wandb/cli/cli.py

```diff
@@ -1586,7 +1586,7 @@ def _list(project, entity):
         wandb.termlog(f"{name} -- versions ({len(aliases)}): {aliases_str}")
 
 
-@job.command(help="Describe a job")
+@job.command(help="Describe a launch job. Provide the launch job in the form of: <entity>/<project>/<job-name>:<alias-or-version>")
```

Comment:
looks like black wants to format this as:

```suggestion
@job.command(
    help="Describe a launch job. Provide the launch job in the form of: <entity>/<project>/<job-name>:<alias-or-version>"
)
```